react/components/EXPERIMENTAL_Table/Toolbar/InputAutocomplete.tsx

```diff
@@ -5,12 +5,15 @@ import { NAMESPACES, ORDER_CLASSNAMES } from '../constants'
 import AutocompleteInput, {
   AutocompleteInputProps,
 } from '../../AutocompleteInput'
+import { useToolbarContext } from './context'
 
 const InputAutocomplete: FC<AutocompleteInputProps> = props => {
+  const { testId } = useToolbarContext()
   return (
     <span
       className={classNames(ORDER_CLASSNAMES.TOOLBAR_CHILD.INPUT, 'w-40')}
-      id={NAMESPACES.TOOLBAR.INPUT_SEARCH}>
+      id={NAMESPACES.TOOLBAR.INPUT_SEARCH}
+      data-testid={`${testId}__input-autocomplete`}>
```

Comment:
there are some files where you put the formatting inside the `data-testid` and others where you create a constant for that, can you do them all the same way?

Reply:
I'm keeping them for not. In an upcoming PR, this will change a bit.